paddle/fluid/operators/math/blas.h

```diff
@@ -46,6 +46,17 @@ namespace paddle {
 namespace operators {
 namespace math {
 
+struct MatDim {
+  int64_t height_;
+  int64_t width_;
+  int64_t stride_{0};
+  int64_t batch_size_{0};
```

Comment:
What is batch_size? Matrix is just a 2D data brick; what is its relationship with *batch*?

Reply:
To further optimize the performance of GEMM, some hardware supports batched matrix computation, I think `MatDim` is a better choice to represent matrix or matrix with batch.
Some introduce from Intel and paper 
https://software.intel.com/en-us/articles/introducing-batch-gemm-operations  
http://www.netlib.org/utk/people/JackDongarra/PAPERS/batched-matrix-comp.pdf.